pkg/apis/garden/v1beta1/helper/helpers.go

```diff
@@ -321,3 +321,47 @@ func IsUsedAsSeed(shoot *gardenv1beta1.Shoot) (bool, *bool, *bool) {
 
 	return useAsSeed, protected, visible
 }
+
+// GetResources returns the cpu and memory requirements for API server based on nodeCount
+func GetResources(nodeCount int) (string, string, string, string) {
```

Comment:
Why is that part of the Garden API helper package? Can we please move it to the package where the kube-apiserver is actually deployed?